projects/1.20/assets/raxiores-dinosaur-mode/raxiores/lang/zh_cn.json

```
@@ -0,0 +1,118 @@
+{
+  "This chest locked.": "这个箱子已上锁。",
+  "subtitles.giganoto.roar": "南方巨兽龙咆哮",

```

```suggestion
  "subtitles.giganoto.roar": "南方巨兽龙：咆哮",
```
字幕格式请参考 [字幕 - Minecraft Wiki](https://zh.minecraft.wiki/w/%E5%AD%97%E5%B9%95)。
诸如此类的问题请在您的译文中全部查明并修正。